commands/deploy_command.go

```diff
@@ -317,6 +316,14 @@ func (c *DeployCommand) Execute(args []string) ExecutionStatus {
 	processBuilder.Parameter("mtaId", mtaID)
 	setModulesAndResourcesListParameters(modulesList, resourcesList, processBuilder, mtaElementsCalculator)
 	c.processParametersSetter(optionValues, processBuilder)
+
+	strategy := GetStringOpt(strategyOpt, optionValues)
+	if len(strategy) > 0 && strategy == "blue-green" {
```

Comment:
Do you need len(strategy) > 0. In Go empty/nil strings are always "".

Reply:
Changed to `if strategy == "blue-green"`